flaskbb/forum/views.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-'''
+"""
```

Comment:
Did you hop on the black train as well?

Reply:
Actually, I did this by hand via search-and-replace. But a day later I installed black ;)